core/src/jsMain/kotlin/Transaction.kt

```diff
@@ -141,7 +163,7 @@ public open class Transaction internal constructor(
             } else if (cursor != null) {
                 val result = trySend(wrap(cursor, channel))
                 when {
-                    result.isSuccess -> if (autoContinue) cursor.`continue`()
+                    result.isSuccess -> if (autoContinue && !finished) cursor.`continue`()
```

Comment:
**info**: I've added this check, so `continue()` is not called after flow was closed. 

Otherwise in the following code:
```kotlin
val returnedValue = database.transaction("customers") {
            objectStore("customers")
                .index("name")
                .openCursor(autoContinue = true)
                .first()
        }
```

Cursor might try to return new value after `first()` and after transaction is finished. This will cause `TransactionInactiveError`.